### PR TITLE
Finished initial work with api

### PR DESCRIPTION
**Pull request by LLkaia**
- added models for article
- added functions to work with MongoDB
- added routs for search and concrete article
- added scripts for scraping

### CODE REVIEW

File: requirements.txt
Comment (AVK1206) [inline requirements.txt:1]: Are u sure u need all of these packages and libraries?
Do u use venv in ur project?

File: server/database.py
Comment (AVK1206) [inline server/database.py:1]: Please add short documentation to the file and to the functions.
The length of code must be shorter than 80 symbols PEP 8 https://peps.python.org/pep-0008/#maximum-line-length.

File: server/models/search_result.py
Comment (AVK1206) [inline server/models/search_result.py:1]: Please add short documentation.

File: server/app.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI
+from server.routes.search_result import router as SearchResultRouter
```
Comment (AVK1206) [inline server/app.py:3]: Please add one blank line between two tdifferent types of import. PEP 8 https://peps.python.org/pep-0008/#imports

File: server/scraper.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    soup_src = BeautifulSoup(page_src.content, 'html.parser')
+
+    laptops = soup_src.find_all('div', class_='listingResult')
+    laptops_data = []
+    for laptop in laptops:
+        dct = dict()
+        dct['link'] = laptop.find('a', class_='article-link').get('href')
+        dct['image'] = laptop.find('img').get('data-pin-media')
+        dct['title'] = laptop.find('h3', class_='article-name').text.strip()
+        dct['author'] = laptop.find('span', attrs={'style': 'white-space:nowrap'}).text.strip()
```
Comment (AVK1206) [inline server/scraper.py:33]: The line of code must not be greater than 79 symbols.

**Review by AVK1206 — COMMENTED** [review]
In general please add documentation to ur files. Follow PEP 8 and PEP 257. 
README file is important to understand what's going on in ur project but ur README file is empty so please update README.

File: requirements.txt
Comment (LLkaia) [reply requirements.txt:1]: ofc i use venv, FastAPI pull all this dependencies

File: server/database.py
Comment (LLkaia) [reply server/database.py:1]: We are working with max line length 120 if i understood properly

File: server/models/search_result.py
Comment (LLkaia) [reply server/models/search_result.py:1]: I m working on it, thanks!

File: server/app.py (hunk 1 above)
Comment (LLkaia) [reply server/app.py:3]: Got it

File: server/scraper.py (hunk 2 above)
Comment (LLkaia) [reply server/scraper.py:33]: We will talk about it next on our next meeting.

File: server/models/search_result.py
Comment (LLkaia) [reply server/models/search_result.py:1]: done

**Review by AVK1206 — COMMENTED** [review]
If u don't need empty init files would be better to delete them.

**Comment by LLkaia** [comment]
> If u don't need empty init files would be better to delete them.

FastAPI's docs recommend to use them (i agree that they are not necessary in python now)

**Review by VovaD199 — APPROVED** [review]

**Pull request merged**